### ISSUE TITLE
Consolidate documentation between r2d and binder?

### ISSUE THREAD

**Issue by jzf2101** [jupyterhub/binder#119]
We have some duplicate documentation on mybinder's read the docs and repo2docker's read the docs.  Do we want that? Should we consolidate and link?

**Comment by betatim** [comment]
I'd keep the documentation on how to build different things in the r2d documentation and link to the appropriate docs from the mybinder.org documentation.

**Comment by choldgraf** [comment]
that's what I've tried to do here:

https://github.com/jupyterhub/binder/pull/116/files#diff-684a0ceb2878d48b19612ca360e277efR187

I agree we don't want them totally overlapping!

**Comment by jzf2101** [comment]
Sorry I'm a bit confused by the diff since it seems as if it were taken from the r2d docs?

**Comment by choldgraf** [comment]
yep - when `conf.py` runs it pulls the latest version of the `howto` files and the `config_files` pages from the repo2docker documentation.

**Comment by choldgraf** [comment]
I just added those files to `.gitignore` so ti's less confusing. They'll automatically be pulled when the docs are built, but won't be part of the repository itself

**Comment by choldgraf** [comment]
the content will be duplicated between the repos, yeah - I think that the target user for `docs.mybinder.org` and `repo2docker` is different enough that it's fine duplicating the content. What do you think?

**Comment by jzf2101** [comment]
I don't know.  I assume that the target audience is similar because you can use repo2docker to do local testing for binder.

**Comment by choldgraf** [comment]
using binder requires no background in docker, images, or even programming (plenty of software carpentries use binder to serve environments)...repo2docker on the other hand is more for people who know what a repository is, what a docker image is, what jupyter is, etc
